### DISCUSSION TITLE
using dd images from real Amiga in amiberry?

### DISCUSSION THREAD

**Discussion by Juppstein** [BlitterStudio/amiberry#1266] [Q&A] [answered]
I made disk images via dd (on linux) from the SD card amiga paritions used in a real Amiga. I am able to attach those images directly in FS-UAE and boot successfully to Workbench. Is there a way to use them in amiberry as well? Just renaming them to .hdf and add them via "Add Hardfile" does not seem to work. amiberry boots into a guru.

**Comment by solskogen** [comment, marked as answer]
This should just work. Can you do a `file` on that hdf? You can try adding it as a IDE (change Controller to A600/A1200/A4000 IDE)
